content/docs/user-guide/experiment-management/index.md

```diff
@@ -35,6 +41,8 @@ meaningful measures for the experimental results.
 
 ## Experiments
 
+> Note: these features require a Git-enabled DVC <abbr>repository</abbr>.
```

Comment:
do we actually need this note? does it apply to the whole experiment management?

Git-enabled DVC repo - still too complicated to my mind.

I would move the note one paragraph below (do not start with some notes that are not even clear if they are relevant for me) and would make it other way around:

`dvc exp` commands won't work if you don't use Git to version control your code

Reply:
Moved down and rephrased for now. I don't know if we need the note. Most realistic uses of DVC need Git I suppose so maybe we should only mention that Git is NOT required in the few that dont? I.e. basic data management with `init --no-scm`.

> use Git to version control your code

This may also be misleading though? It's not just code but config files (importantly dvc metafiles).